packages/components/src/components/Slider/styles/slider.module.scss

```diff
@@ -0,0 +1,41 @@
+
+.SliderRoot {
```

Comment:
Camel case the css variables

Reply:
yep, noticed that, no worries these styles are placeholders